src/app/activity/bounty/Create.ts

```diff
@@ -13,13 +13,28 @@ import { PaidStatus } from '../../constants/paidStatus';
 import { Activities } from '../../constants/activities';
 import DMPermissionError from '../../errors/DMPermissionError';
 import { ComponentType, ModalInteractionContext, TextInputStyle } from 'slash-create';
+import { BountyCollection } from '../../types/bounty/BountyCollection';
 
 export const createBounty = async (createRequest: CreateRequest): Promise<any> => {
     Log.debug('In Create activity');
 
     if (createRequest.isIOU) {
         await finishCreate(createRequest, null, 'IOU for work already done', new Date(), null);
+    } else if (createRequest.templateId) {
+        // If we have a templateId, this came from the background cron job creating repeating bounties. Load everything and go create
+        const db: Db = await MongoDbUtils.connect('bountyboard');
+        const dbBounty = db.collection('bounties');
+        const template: BountyCollection = await dbBounty.findOne({'_id': new mongo.ObjectId(createRequest.templateId)});
```

Comment:
@tescher did you consider creating a collection for bounty templates? Presence of templateId is enough to identify a document as a bounty template - but might be worth if you see the future need for a one to many relationship between templates and created bounties.